Command/DumpSitemapsCommand.php

```diff
@@ -56,7 +58,7 @@ public function __construct(RouterInterface $router, DumperInterface $dumper, $d
      */
     protected function configure()
     {
-        $this->setName('presta:sitemaps:dump')
+        $this->setName(self::$defaultName)
```

Comment:
Correct me if i'm wrong, but calling `setName()` is not required since `$defaultName` is filled.

Reply:
Yes, you are right, updated :)